### PR TITLE
Add command line arguments to monero-wallet-rpc.reference.md and Add Dark Mode

### PR DESCRIPTION
**Pull request by elibroftw**
- Added the command line arguments shown when `--help`  is used on `monero-wallet-rpc`.
- Added command line examples
- Added trouble shooting section to mention that an unresponsive or *dead* RPC may be due to the wallet file not being not synchronized 
  - Added solution and ways to mitigate this problem
  - Added dark mode (#121212 for bg, and Monero Orange for accent)

### CODE REVIEW

File: docs/interacting/monero-wallet-rpc-reference.md [hunk 1]
```diff
@@ -35,85 +36,231 @@ curl \
...
+`~/bin/monero-wallet-rpc --rpc-bind-port 28088 --wallet-file ~/monero/wallets/main/main --password {walletPWD} '--rpc-login', monero:{uncommonPWD} --log-file logs/monero-wallet-rpc.log --max-log-files 2 --trusted-daemon --non-interactive`
+
+- If only incoming transactions matter, ideally the wallet should be a view-only to prevent abuse
+- `--rpc-login` should be used in production to protect against any network attacks. An uncommon password protects against the case where the RPC port is open to the public
+
+#### Windows (Development Example)
+
+`"C:\Program Files\Monero GUI Wallet\monero-wallet-rpc.exe" --rpc-bind-port 28088 --wallet-file C:\monero\wallets\main\main --password {walletPWD} --daemon-address http://node.supportxmr.com:18081 --untrusted-daemon --disable-rpc-login --non-interactive`
+
+- Specifying `--untrusted-daemon` is not neccesary but it's there to tell yourself that the daemon is untrusted and that commands requiring a trusted daemon will be disabled
```
Comment (serhack) [inline docs/interacting/monero-wallet-rpc-reference.md]: typo: `neccesary` => `necessary`.

Comment (elibroftw) [reply docs/interacting/monero-wallet-rpc-reference.md]: Fixed

**Comment by elibroftw** [comment]
Also to whomever merges this, can you please add https://monerodocs.org/ to the repository's website field?

**Comment by CryptoGrampy** [comment]
Tested the dark mode on Chromium and Firefox.  Looks amazing!  Nice job @elibroftw

**Comment by qertoip** [comment]
Thanks so much for your PR! This is very much welcome.

Some background: this specific page (monero-wallet-rpc-reference) was originally auto-generated by transforming a relevant page from getmonero. As such it is lacking in consistency with other pages.

Kindly please:
* Adjust your new content to be consistent with reference for other programs (like monerod) in terms of chapters and subchapters naming and order, how examples are presented (no full path etc, example values instead of interpolation, etc)
* Use default ports, in this case 18081. I recognize the existing auto-generated content already uses non-default port and that was misleading. We can fix the existing content in a separate PR in that regard.
* Move the units information further down as it is likely **not** the most important information on this page.

On the organization side please:
* Extract visual changes into a separate PR.
* Squash your commits into a single one regarding monero-wallet-rpc-reference.

Thanks again!

**Comment by elibroftw** [comment]
I don't agree with the full paths point. By providing a full path, we are sharing knowledge on where the wallet-rpc would reside on a server and where it would be on a development machine. The former is a recommendation to download and store monero binaries in ~/bin and the latter tells the developer the installation location of the monero binaries on windows, saving them the minute or so it takes to hunt it down themselves.

**Comment by qertoip** [comment]
Thanks for your comment!

The paths vary greatly across operating systems, Linux distributions, conventions and user preferences. I like the docs to be agnostic regarding the paths.

The default installation path under Windows can be covered on the Interacting / Overview page.

**Comment by elibroftw** [comment]
"Use default ports, in this case 18081"

The wallet-rpc does not have a default port. 18081 is for the daemon not the wallet-rpc. 28088 is best to use in order to stay consistent with the other docs.

**Comment by qertoip** [comment]
@elibroftw you are right, the 18081 is daemon RPC, scratch my remark regarding the ports.

**Comment by elibroftw** [comment]
Going to create new PR's

**Comment by elibroftw** [comment]
#39 and #40

**Pull request closed without merging**